eu_dgc_v1_schema.json

```diff
@@ -40,7 +61,7 @@
                         "properties": {
                             "t": {
                                 "title": "Identifier type",
-                                "description": "The type of identifier (viz. VS-2021-04-08) PPN = Passport NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code) CZ = Citizenship card HC = Health Card number etc.",
+                                "description": "The type of identifier (viz. VS-2021-04-14) PPN = Passport NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code) CZ = Citizenship card HC = Health Card number etc.",
```

Comment:
See suggested change in #23 as that would help implementors getting this right. That gives the possibility to say "Swedish passport number 123" or "Danish health card 456" and similar.